packages/amplify-e2e-core/src/categories/custom.ts

```diff
@@ -42,3 +47,10 @@ export function buildCustomResources(cwd: string) {
       });
   });
 }
+
+export const useLatestExtensibilityHelper = (projectRoot: string, customResourceName: string) => {
+  const packageJsonPath = path.join(projectRoot, 'amplify', 'backend', 'custom', customResourceName, 'package.json');
+  const packageJson: Record<string, Record<string, string>> = JSONUtilities.readJson(packageJsonPath);
+  packageJson.dependencies['@aws-amplify/cli-extensibility-helper'] = 'latest';
```

Comment:
This is accompanied by starting verdaccio on VM (we already pull verdaccio cache). In that case latest is either from source (if there was any change) or from npm otherwise.

Reply:
understood , is this needed if we have set it with major version to simulate customer pkg.json?

Reply:
Yes. publish to local registry step does this https://github.com/aws-amplify/amplify-cli/blob/f03d170ae01d42a934f7b1392762f4d231ab687a/.circleci/publish.sh#L38-L41 .

But. The version that lands in registry is "tagged" for example `2.4.11-rc.021a139b2.0` . Soo '^2.0.0' won't match it. And figuring exact version is PITA, plus the package is not always published to local registry (if there's no change there or it's dependencies). `latest` covers all cases with minimum energy.

![image](https://user-images.githubusercontent.com/5849952/221011641-9d26fdce-577d-469b-8615-fdef6ac09ba6.png)